examples/mongodbatlas_federated_database_instance/azure/README.md

```diff
@@ -0,0 +1,94 @@
+# Example - MongoDB Atlas Federated Database Instance with Azure Blob Storage and MongoDB Cluster as storage databases
+
+This project aims to provide an example of using [MongoDB Atlas Federated Database Instance](https://www.mongodb.com/docs/atlas/data-federation/overview/).
+
+## Dependencies
+
+* Terraform MongoDB Atlas Provider v1.39.0
+* A MongoDB Atlas account
+* An Azure account
+
+```
+Terraform v1.39.0
++ provider registry.terraform.io/mongodb/mongodbatlas v1.39.0
+```
+
+## Usage
+
+**1\. Ensure your Azure credentials are set up.**
+
+1. Install the Azure CLI by following the steps from the [official Azure documentation](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+2. Run the command `az login` and this will take you to the default browser and perform the authentication.
```

Comment:
Run the command `az login`. This takes you to the default browser where you can authenticate with Azure.